cookbooks/aws-parallelcluster-config/recipes/update.rb

```diff
@@ -21,8 +21,14 @@
   end
 end
 
+ruby_block "backup storages mappings" do # backup shared storages mapping file to used for update and rollback
+  block do
+    ::FileUtils.cp_r(node['cluster']['shared_storages_mapping_path'], node['cluster']['previous_shared_storages_mapping_path'], remove_destination: true)
```

Comment:
Minor: I think `previous_*` does not explicitly clarify the goal of that file.
If the goal is to store the configuration before the actual execution of an update/rollback, what about naming it `pre_update_*`

Reply:
For cluster config, we name it previous-cluster-config.yaml, I think it is better to be aligned with that

Reply:
Got it, let's keep them aligned.